src/engraving/layout/v0/tlayout.cpp

```diff
@@ -3602,28 +3602,32 @@ void TLayout::layout(NoteDot* item, LayoutContext&)
 
 void TLayout::layout(Ornament* item, LayoutContext& ctx)
 {
+    if (item->showCueNote()) {
+        return;
+    }
+
     double _spatium = item->spatium();
     double vertMargin = 0.35 * _spatium;
     static constexpr double ornamentAccidentalMag = 0.6; // TODO: style?
 
-    if (!item->showCueNote()) {
-        for (size_t i = 0; i < item->accidentalsAboveAndBelow().size(); ++i) {
-            bool above = (i == 0);
-            Accidental* accidental = item->accidentalsAboveAndBelow()[i];
-            if (!accidental) {
-                continue;
-            }
-            accidental->computeMag();
-            accidental->setMag(accidental->mag() * ornamentAccidentalMag);
-            layout(accidental, ctx);
-            Shape accidentalShape = accidental->shape();
-            double minVertDist = above ? accidentalShape.minVerticalDistance(item->bbox()) : Shape(item->bbox()).minVerticalDistance(
-                accidentalShape);
-            accidental->setPos(-0.5 * accidental->width(), above ? (-minVertDist - vertMargin) : (minVertDist + vertMargin));
+    for (size_t i = 0; i < item->accidentalsAboveAndBelow().size(); ++i) {
+        bool above = (i == 0);
+        Accidental* accidental = item->accidentalsAboveAndBelow()[i];
+        if (!accidental) {
+            continue;
         }
-        return;
+        accidental->computeMag();
+        accidental->setMag(accidental->mag() * ornamentAccidentalMag);
+        layout(accidental, ctx);
+        Shape accidentalShape = accidental->shape();
+        double minVertDist = above ? accidentalShape.minVerticalDistance(item->bbox()) : Shape(item->bbox()).minVerticalDistance(
+            accidentalShape);
+        accidental->setPos(-0.5 * accidental->width(), above ? (-minVertDist - vertMargin) : (minVertDist + vertMargin));
     }
+}
 
+void TLayout::layoutTrillCueNote(Ornament* item, LayoutContext& ctx)
```

Comment:
With this name, it makes more sense to have Trill* as an argument (instead of Ornament*)

Reply:
Maybe rename this function to layoutOrnamentCueNote? Or add something like:

```
TLayout::layoutTrillCueNote(Ornament* item, LayoutContext& ctx) {
    IF_ASSERT_FAILED(item->isTrillType()) {
        return;
    }
    
    ...
}
```